forge-gui/res/cardsfolder/upcoming/ashiok_wicked_manipulator.txt

```diff
@@ -3,12 +3,12 @@ ManaCost:3 B B
 Types:Legendary Planeswalker Ashiok
 Loyalty:5
 R:Event$ PayLife | ActiveZones$ Battlefield | ValidPlayer$ You | ReplaceWith$ ExileTop | Amount$ LEY | Description$ If you would pay life while your library has at least that many cards in it, exile that many cards from the top of your library instead.
-SVar:ExileTop:DB$ Dig | DigNum$ X | ChangeNum$ All | DestinationZone$ Exile
+SVar:ExileTop:DB$ DigUntil | Amount$ X | Valid$ Card.NotDefinedReplacedSimultaneousETB | NoMoveRevealed$ True | FoundDestination$ Exile

```

Comment:
You need to check if your DigUntil change does alter how the Cards are Exiled

Ashiok should still exile them at once, not in sequence

Reply:
yea that part is ok
only if RevealedDestination$ was also Exile DigUntil logic would think it's doing "Exile Until"